src/Main.java

```diff
@@ -0,0 +1,8 @@
+public class Main {
+    int a;
+    public static void main(String[] args) {
+        A a =new A();
+        B b = new B();
+        C c = new C();
```

Comment:
@kuchmei I said you that only one instance of class C is needed to show initialization order
Please change and rewrite console output